app/views/common/os_selection/_pxe_loader.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= fields_for item do |f| %>
+  <% selected_attr = f.object.new_record? && item.hostgroup.pxe_loader.nil? ? { :selected => @operatingsystem.preferred_loader } : {} -%>
+  <%= selectable_f f, :pxe_loader,
+    @operatingsystem.available_loaders.to_a, {
+      :include_blank => blank_or_inherit_f(f, :pxe_loader),
+      :selected => item.pxe_loader
+    }.merge(selected_attr),
+    :label => _('PXE Loader'),
+    :help_inline => popover(_("DHCP filename option"), _("TFTP filename to use, select one of")  + ' ' + @operatingsystem.available_loaders.values.join(', ') + _('or set to blank for Non-PXE method (e.g. iPXE)'), :data => { :placement => 'top' }) %>
+<% end %>
```

Comment:
The `selected_attr` is a "recommendation" when hostgroup does not define `pxe_loader`. It gives "sane default" when not using hostgroup or not set by hostgroup.

But I can't get hostgroup selection working - when I set a hostgroup that has `pxe_loader` set, it's left blank. Do I need to change something in our spaghetti javascript code in order to have that?